docs/upgrade.html

```diff
@@ -47,14 +47,16 @@
 
 <p><b>Note:</b> Bumping the protocol version and restarting can be done any time after the brokers were upgraded. It does not have to be immediately after.
 
-<ol>
-    <li>Upgrading a Kafka Streams Applications:
-        <ul>
-            <li>You need to recompile your code. Just swapping the jar file will not work and will break your appliation.</li>
-            <li>If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed.</li>
-        </ul>
-    </li>
-</ol>
+<h5><a id="upgrade_1020_streams" href="#upgrade_1020_streams">Upgrading a Kafka Streams Application</a></h5>
+<ul>
+    <li> You need to recompile your code. Just swapping the jar file will not work and will break your application. </li>
+    <li> <code>KStreamBuilder#table()</code> always requires a store name. </li>
+    <li> <code>KTable#through()</code> always requires a store name. </li>
+    <li> If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed. </li>
```

Comment:
"you will need to update this code" -- how?  There's no info here.  IMHO there are two main scenarios:

1. The user has configured a different but built-in timestamp extractor.  To update/upgrade the application, the user must only configure the app to use one of the new built-in extractors (= it's mostly about renaming a config option setting).
2. The user implemented a custom timestamp extractor.  What needs to be done here?

Also, we should at least link back to the Changes section (see above in the PR) where we list/explain the new built-in timestamp extractors, and the new default pick.